lib/link_thumbnailer/scrapers/opengraph/image.rb

```diff
@@ -20,7 +21,11 @@ def value
           end
 
           def model
-            nodes.map { |n| modelize(n, n.attributes['content'].to_s) }
+            nodes.map do |n|
+              if LinkThumbnailer::URI.new(n.attributes['content']).valid?
```

Comment:
Yea, that looks better. I've updated the code and added some tests for the `LinkThumbnailer::URI` class